src/tasks/FishingTask.py

```
@@ -33,12 +33,31 @@ def __init__(self, *args, **kwargs):
         self.default_config.update(
             {
                 "循环次数": 1,
+                "控条方式": "状态切换",
+                "离散按键倍数": 1.0,
             }
         )
+        self.config_description.update(
+            {
+                "控条方式": "状态切换：快，持续按住，平滑流畅\n离散按键：慢，防止过冲",
```

Ruff (0.15.12)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)